### PR TITLE
Fix Boolean parameters in models.

### PR DESCRIPTION
**Pull request by davidporos92**
refs #100

### CODE REVIEW

File: app/engine/Form/Element/Select.php [hunk 1]
```diff
@@ -128,6 +128,7 @@ public function setValue($value, $escape = false)
 
         $elementOptions = $this->getOption('elementOptions', []);
         foreach ($value as $currentValue) {
+            if ($currentValue !== null && !is_string($currentValue) && !is_int($currentValue)) $currentValue = (string)$currentValue;
```
Comment (sergeyklay) [inline app/engine/Form/Element/Select.php]: Could you please fix PSR error here?

Comment (davidporos92) [inline app/engine/Form/Element/Select.php]: Do you mean the curly braces of the if statement?

Comment (lantian) [inline app/engine/Form/Element/Select.php]: Yes

**Comment by lantian** [comment]
Thanks

**Comment by davidporos92** [comment]
You are welcome. I am happy to help. :)

**Pull request merged**